Resources/translations/SonataAdminBundle.pt_BR.xliff

```diff
@@ -164,11 +164,11 @@
             </trans-unit>
             <trans-unit id="flash_lock_error">
                 <source>flash_lock_error</source>
-                <target>Outro usuário modificou o item "%name%". Por favor, %link_start%clique aqui%link_end% to para recarregar a página e aplicar as mudanças novamente.</target>
+                <target>Outro usuário modificou o item "%name%". Por favor, %link_start%clique aqui%link_end% para recarregar a página e aplicar as mudanças novamente.</target>
             </trans-unit>
             <trans-unit id="flash_batch_delete_success">
                 <source>flash_batch_delete_success</source>
-                <target>Itens solucionados foram deletados com sucesso.</target>
+                <target>Itens selecionados foram deletados com sucesso.</target>
```

Comment:
This translation is specially important! Previously it said "solved items ..." and now it correctly says "selected items ..."